modules/tinymce/src/plugins/link/test/ts/browser/AttributesPostprocessTest.ts

```diff
@@ -0,0 +1,64 @@
+import { ApproxStructure, FocusTools, Waiter } from '@ephox/agar';
+import { describe, it, before, after } from '@ephox/bedrock-client';
+import { Fun } from '@ephox/katamari';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/link/Plugin';
+
+import { TestLinkUi } from '../module/TestLinkUi';
+
+describe('browser.tinymce.plugins.link.AttributesPostprocess', () => {
+  const newRel = Fun.constant('noopener noreferrer');
```

Comment:
Why the constant function if it's invoked to get the value out at both places? Some refactor that was done and this was left there? You could just use a string constant? Function constants is useful if you need to pass it to some function that is higher order.